lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -58,16 +58,22 @@ def header
         end
       end
 
-    private
-
-      attr_reader :content_item, :current_path
-
-      def first_step_nav
+      def get_active_step_nav
```

Comment:
This should really be two functions, one that returns the active step nav, and the other that checks if there is an active step nav.